open-company/open-company-web
# Missing components tests

card: https://trello.com/c/5MD9AsJx

to test:
- run `lein test-all!`
- make sure tests for company are run (ie these components appear in the log: company, headcount-seciton...)
